### PR TITLE
cnf network: Fix sriovmetrics Tests on Intel NICs

### PR DESCRIPTION
**Pull request by ajaggapa**
- Create SriovNodePolicies together
- Clear ARP table on switch before executing each Vfio test case
- Increase wait time @line449 for flaky scenarios

### CODE REVIEW

File: tests/cnf/core/network/sriov/tests/metricsExporter.go [hunk 1]
```diff
@@ -397,44 +403,56 @@ func defineDPDKPod(role, devType, worker string) *pod.Builder {
...
-	Expect(err).ToNot(HaveOccurred(), "Failed to create SriovNetworkNodePolicy")
+	err := netenv.WaitForSriovAndMCPStable(APIClient, tsparams.MCOWaitTimeout, tsparams.DefaultStableDuration,
+		NetConfig.CnfMcpLabel, NetConfig.SriovOperatorNamespace)
 
-	By(fmt.Sprintf("Creating SriovNetwork %s", testRes.network.Definition.Name))
+	By("Create SriovNetwork and wait for NAD creation")
 
-	_, err = testRes.network.Create()
-	Expect(err).ToNot(HaveOccurred(), "Failed to create SR-IOV network")
+	for _, res := range []testResource{cRes, sRes} {
```
Comment (evgenLevin) [inline tests/cnf/core/network/sriov/tests/metricsExporter.go:420]: Just suggestion:
You can create policy and network in the same itteration. And then wait for MCP stable and nad verify that nad has been created

File: tests/cnf/core/network/sriov/tests/metricsExporter.go [hunk 2]
```diff
@@ -488,3 +506,19 @@ func setMetricsExporter(flag bool) {
 	_, err = defaultOperatorConfig.Update()
 	Expect(err).ToNot(HaveOccurred(), "Failed to update metricsExporter flag in default Sriov Operator Config")
 }
+
+func clearClientServerMacTableFromSwitch() {
+	switchCredentials, err := sriovenv.NewSwitchCredentials()
+	Expect(err).ToNot(HaveOccurred(), "Failed to get switch credentials")
+
+	jnpr, err := cmd.NewSession(switchCredentials.SwitchIP, switchCredentials.User, switchCredentials.Password)
+	Expect(err).ToNot(HaveOccurred())
```
Comment (evgenLevin) [inline tests/cnf/core/network/sriov/tests/metricsExporter.go:515]: Please descriptions to all Expects in this func

**Review by evgenLevin — COMMENTED** [review]
2 minor comments

File: tests/cnf/core/network/sriov/tests/metricsExporter.go (hunk 1 above)
Comment (ajaggapa) [reply tests/cnf/core/network/sriov/tests/metricsExporter.go:425]: done

File: tests/cnf/core/network/sriov/tests/metricsExporter.go (hunk 2 above)
Comment (ajaggapa) [reply tests/cnf/core/network/sriov/tests/metricsExporter.go]: done

**Review by evgenLevin — APPROVED** [review]
/lgtm

**Comment by gkopels** [comment]
lgtm

**Review by gkopels — APPROVED** [review]

**Pull request merged**